### ISSUE TITLE
[BUG] Does this library prevent inspection of accessibilityLabels, IDs, and Test IDs?

### ISSUE THREAD

**Issue by philipfong** [gunnartorfis/sonner-native#146]
Because this library acts like an overlay on our mobile app, it seems to prevent the inspection of IDs on basic components like inputs boxes and buttons. Is there a way to incorporate this into our app while being able to view these type of props? We are looking to build automated tests in Maestro or Detox but are unable to without disabling this component.

**Comment by gunnartorfis** [comment]
If you use element inspector, does it look like this issue report? https://github.com/gunnartorfis/sonner-native/issues/114

Could you post your setup? How and where the Toaster is rendered especially.

**Comment by philipfong** [comment]
Hi @gunnartorfis -- yes that is pretty much the exact side effect that we're seeing.

Here is what looks to be the relevant code in our App.tsx 

```
    <GestureHandlerRootView style={styles.layout}>
      <KeyboardProvider>
        <SafeAreaProvider>
          <QueryClientProvider client={queryClient}>
            <ModalProvider stack={modalStack}>
              <RootNavigator />
            </ModalProvider>
          </QueryClientProvider>
          <Toaster position="bottom-center" />
        </SafeAreaProvider>
      </KeyboardProvider>
    </GestureHandlerRootView>
```

If I remove the line with the Toaster tag I'm able to inspect elements again.

Please advise as we'd like to keep the toaster messages in place. Our other alternative is to remove this library conditionally for testing purposes.

**Comment by gunnartorfis** [comment]
@philipfong Is the NavigationContainer defined in the RootNavigator component? 

Can you try moving the <Toaster /> outside the KeyboardProvider? 

```ts
<GestureHandlerRootView style={styles.layout}>
      <KeyboardProvider>
        <SafeAreaProvider>
          <QueryClientProvider client={queryClient}>
            <ModalProvider stack={modalStack}>
              <RootNavigator />
            </ModalProvider>
          </QueryClientProvider>
        </SafeAreaProvider>
      </KeyboardProvider>
      <Toaster position="bottom-center" />
    </GestureHandlerRootView>
```

**Comment by philipfong** [comment]
Tried that and it generates an error related to `SafeAreaProvider`.

**Comment by gunnartorfis** [comment]
Keep the SafeAreaProvider around the Toaster, try that. This is roughly the exact same setup as my personal app uses:

```ts
<GestureHandlerRootView style={styles.layout}>
        <SafeAreaProvider>
      <KeyboardProvider>
          <QueryClientProvider client={queryClient}>
            <ModalProvider stack={modalStack}>
              <RootNavigator />
            </ModalProvider>
          </QueryClientProvider>
        </SafeAreaProvider>
      <Toaster position="bottom-center" />

      </KeyboardProvider>
    </GestureHandlerRootView>
```

**Comment by vargas-arthur** [comment]
@gunnartorfis I have the same problem. My setup is really similar to @philipfong.

Tried putting Toaster in different places but none worked.

**Comment by gunnartorfis** [comment]
@vargas-arthur could you share how you render the Toaster? A minimum repro GitHub repo would be highly appreciated.

**Comment by gunnartorfis** [comment]
@philipfong @vargas-arthur 

https://github.com/gunnartorfis/sonner-native/issues/114#issuecomment-2427297469

Would be great to know if this works for you guys too!

**Comment by philipfong** [comment]
We've tried moving <Toaster> up and down our entire hierarchy without success. I've tried to move this right outside the NavigationContainer as the other issue recommends, but we're still not seeing elements.

I suspect maybe our usage of `ModalProvider` in `react-native-modalfy` is causing a problem? But I'm not too familiar with that library.

**Comment by gunnartorfis** [comment]
@philipfong I'm really confused about this as well. I recommend creating a minimum reproducible example, that might both help you narrow down the culprit and me fix it if it is an issue on my end.

**Comment by vargas-arthur** [comment]
Hey @gunnartorfis , I will try to do that this week and send it to you.

**Comment by gunnartorfis** [comment]
@vargas-arthur @philipfong  could you try out the `0.16.0-beta.0` release? There we're only showing the FullWindowOverlay conditionally if there are any toasts visible.

**Comment by gunnartorfis** [comment]
Fixed via #153

**Comment by vargas-arthur** [comment]
Hey @gunnartorfis,
Apologies for the delayed response—I completely forgot to reply earlier!

I’ve tested the new release, and it works perfectly. Thank you so much for the quick fix and your prompt support. Greatly appreciated! 😄
